stress/frontend/config/stressTestInfra.conf

```diff
@@ -0,0 +1,3 @@
+sweetSpot = 14 //maximum throughput the backend can handle
```

Comment:
14 what?  requests per second, per minute?  Number of workers?  Can we get more of description on this